embassy-stm32/src/pwr/mod.rs

```diff
@@ -1,4 +1,5 @@
 #[cfg_attr(any(pwr_h7, pwr_h7smps), path = "h7.rs")]
+#[cfg_attr(not(any(pwr_h7, pwr_h7smps)), path = "none.rs")]
```

Comment:
why not do `#[cfg(pwr)]` in lib.rs so that the `pwr` module is never imported?

Reply:
`lib.rs` already has `#[cfg(pwr)]`, but it's enabled for the F4 (since it has a pwr block configured), but I use the reg block directly instead of creating an abstraction in this folder, because it's just two register writes...

Reply:
Hmm, but it would probably be better to use `#[cfg_attr(pwr_f4)` instead of `not(any(`